packages/flutter/lib/src/material/input_decorator.dart

```diff
@@ -726,7 +726,9 @@ class _RenderDecoration extends RenderBox with SlottedContainerRenderObjectMixin
        _expands = expands,
        _material3 = material3;
 
-  static const double subtextGap = 8.0;
+  // TODO(bleroux): consider defining this value as a Material token and making it
+  // configurable by InputDecorationTheme.
```

Comment:
This is probably a good idea. I've seen issues where people wanted to be able to control the various fine spacing in InputDecorator.